src/cmd/linuxkit/moby/opts.go

```diff
@@ -0,0 +1,10 @@
+package moby
+
+type BuildOpts struct {
```

Comment:
If the struct is to be part of an API, I would document it too.

Reply:
Fair enough.